daemon/gamemode-context.c

```diff
@@ -87,6 +87,8 @@ struct GameModeContext {
 
 	struct GameModeCPUInfo *cpu; /**<Stored CPU info for the current CPU */
 
+	GameModeIdleInhibitor* idle_inhibitor;
```

Comment:
This fails the clang format check, because the `*` should be beside the variable name.